### PR TITLE
set up the release notes page and include the release notes for v0.1.8

### PR DESCRIPTION
**Pull request by hengm3467**
Thanks a lot for your contributions to the v0.1.8 release notes. I removed two sections ("Refactoring" and "Performance") from the initial draft, as the details in these two sections are invisible to users (users are people who want to try out / test RisingWave but don't care about how it is implemented internally). Please leave any comments that you may have. Thanks!

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+This version was released on May 14, 2022. 
+
+## Main changes
+
+### SQL features
+
+* Support SQL functions: `concat_ws()`, `abs()`, `round()`, `ceil()`, `floor()`. ([#2589](https://github.com/singularity-data/risingwave/pull/2589), [#2531](https://github.com/singularity-data/risingwave/pull/2531))
+* Support casts from number types (`smallint`, `integer`, `bigint`, `numeric`, `real`, and `double precision`) to `varchar` using the `to_string` method. ([#2522](https://github.com/singularity-data/risingwave/pull/2522))
+* Support creating a table with nested columns. ([#2434](https://github.com/singularity-data/risingwave/pull/2434)) 
+* Supports non-literals in the `IN` operator. ([#2588](https://github.com/singularity-data/risingwave/pull/2588))
```
Comment (Sunt-ing) [inline docs/release-notes.md]: Support

**Review by Sunt-ing — COMMENTED** [review]
LGTM

File: docs/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Support the Datagen Source Connector, which can be used to generate mock data for testing purposes. ([#2737](https://github.com/singularity-data/risingwave/pull/2737))
+
+### Observability
+
+* Add metrics for these components in Grafana.
+    * Compaction and object-store  ([#2573](https://github.com/singularity-data/risingwave/pull/2573), [#2761](https://github.com/singularity-data/risingwave/pull/2761))
+    * Iterator and cache ([#2709](https://github.com/singularity-data/risingwave/pull/2709))
+    * Streaming exchange service ([#2906](https://github.com/singularity-data/risingwave/pull/2906))
+
```
Comment (tabVersion) [inline docs/release-notes.md:37]: plus adding throughput of source for each partition [#3505](https://github.com/singularity-data/risingwave/pull/3505)

**Review by tabVersion — APPROVED** [review]
rest lgtm

**Pull request merged**